packages/native/src/lib/ElementAssertion.ts

```diff
@@ -113,4 +140,24 @@ export class ElementAssertion extends Assertion<ReactTestInstance> {
     const { parent } = element;
     return parent !== null && (this.isElementDisabled(element) || this.isAncestorDisabled(parent));
   }
+
+  private isElementVisible(element: ReactTestInstance): boolean {
+    const { type } = element;
+    const elementType = type.toString();
+    if (elementType === "Modal") {
+      return Boolean(element.props && element.props.visible);
+    }
```

Comment:
[nit] This code can be simpler 🙂 

```suggestion

    if (type.toString() === "Modal") {
      return Boolean(element.props?.visible);
    }
```